tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,92 @@
+# Quickstart
+
+In this guide we're going to give a 5 minute introduction to the essential concepts in Eve. If you've never used Eve before, you're in the right place.
+
+## Adding records to Eve
+
+Eve represents data as records, which are key value pairs attached to a unique ID. Eve programs are made up of small composable blocks that search for and create records. Let's start with a block that adds a record to Eve:
+
+```eve
+commit
+  [#greeting text: "hello world"]
+```
+
+This record is tagged `#greeting`, and has an attribute "text" with the value "hello world". Although tags have a special syntax, they are attributes like any other (i.e. #foo = tag: "foo"). We encourage you to classify groups of related records with tags.
+
+## Finding records in Eve
+
+We can search for records, then bind new records to those that are found. Eve finds every record matching the supplied patterns, and then binds records dependent on them. If no records match the search, then the block does not run at all. A block will only run if every pattern in a search matches at least one record. Let's search for the `#greeting` we just committed, and then display it in a text container:
```

Comment:
the first two sentences feel redundant to me; i prefer the wording of the first. the second sentence brings up the idea of dependency, which doesn't feel like the right idea to evoke. the bound records are by nature related, but i don't know that i'd call them dependent. it just feels like a new concept, but we don't talk about dependency anywhere else. i think we get a simpler and more understandable concept by saying "Eve searches for records that match the supplied patterns, then binds news records to them. If no records match..."